lib/linguist/heuristics.rb

```diff
@@ -68,6 +68,20 @@ def call(data)
     # Common heuristics
     ObjectiveCRegex = /^\s*(@(interface|class|protocol|property|end|synchronised|selector|implementation)\b|#import\s+.+\.h[">])/
 
+
+    CPPRegexType0 = /^\s*#.*include.*<(stdint|cstdint|string|vector|map|list|array|bitset|queue|stack|forward_list|unordered_map|unordered_set|(i|o|io)stream).*>/
```

Comment:
Hello, @Alhadis 

So just confirming:

Did you mean split the regexes over multiple lines:

```
%r {
(...)
}x

```
or suggested capturing groups?

I think that use capturing groups could obfuscate the code and lead to misunderstandings in the future besides the performance cost.